server/src/main/java/org/opensearch/index/search/stats/SearchStats.java

```diff
@@ -130,6 +141,12 @@ private Stats(StreamInput in) throws IOException {
             queryTimeInMillis = in.readVLong();
             queryCurrent = in.readVLong();
 
+            if (in.getVersion().onOrAfter(Version.V_2_10_0)) {
```

Comment:
```suggestion
            if (in.getVersion().onOrAfter(Version.V_3_0_0)) {
```

Otherwise the `main` check won't pass